recipes/threadpool/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(test_package)
+
+set(CMAKE_CXX_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+
```

Comment:
```suggestion
set_property_TARGET(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)

```

Reply:
isn't it the same effect as `set(CMAKE_CXX_STANDARD 11)`? I think it's just matter of styling

Reply:
In this case it would be the same, but what I did is global and what @uilianries suggested only affects the target, which is cleaner and most of the time the right way to do it.

Reply:
Prefer cmake target, it's a best practice recommended by cmake community.